
# Try and Except Statement â€“ Catching Exceptions
# Try and except statements are used to catch and handle exceptions in Python.
# Statements that can raise exceptions are kept inside the try clause and the statements that handle the exception are written inside except clause.


print('hello python',1)
print('hello python',2)
a = 10
b = 30
try:
  print(x)
except:
    print('over come the name error')
print('hello python',3)
print('hello python',4)

# hello python 1
# hello python 2
# over come the name error
# hello python 3
# hello python 4
